src/Network/Network/AzureFirewall/ApplicationRuleCollection/NewAzureFirewallApplicationRuleCollectionCommand.cs

```diff
@@ -49,7 +49,6 @@ public class NewAzureFirewallApplicationRuleCollectionCommand : NetworkBaseCmdle
         [ValidateSet(
             MNM.AzureFirewallRCActionType.Allow,
             MNM.AzureFirewallRCActionType.Deny,
-            MNM.AzureFirewallRCActionType.Alert,
```

Comment:
This is a breaking change. `Alert` was added in [Network 1.6.0](https://github.com/Azure/azure-powershell/blob/master/src/Network/Network/ChangeLog.md#version-160), and cannot be removed outside of a breaking change release. Please revert.